extensions/amp-story/1.0/bookend/amp-story-bookend.js

```diff
@@ -505,21 +506,31 @@ export class AmpStoryBookend extends AMP.BaseElement {
     this.assertBuilt_();
     this.isBookendRendered_ = true;
 
-    this.renderComponents_(bookendConfig.components);
+    return this.renderComponents_(bookendConfig.components);
   }
 
   /**
    * @param {!Array<!../bookend/bookend-component.BookendComponentDef>} components
+   * @return {!Promise<?../localization.LocalizationService>}
    * @private
    */
   renderComponents_(components) {
     dev().assertElement(this.bookendEl_, 'Error rendering amp-story-bookend.');
-    const fragment = BookendComponent
-        .buildElements(components, this.win.document);
-    const container = dev().assertElement(
-        BookendComponent.buildContainer(this.getInnerContainer_(),
-            this.win.document));
-    this.mutateElement(() => container.appendChild(fragment));
+
+    return Services
+        .localizationServiceForOrNull(this.win).then(localizationService => {
+          const bookendEls = BookendComponent
+              .buildElements(
+                  components, this.win.document, localizationService);
+          const container = dev().assertElement(
+              BookendComponent.buildContainer(this.getInnerContainer_(),
+                  this.win.document));
+          this.mutateElement(() => container.appendChild(bookendEls));
+          return localizationService;
```

Comment:
Re: above comment
maybe we could simply return `bookendEls` instead here (or return nothing, if we don't have any use for it)